oesukam/andela-sendit
# #162459976 Apply pitch feedback

#### What does this PR do?
Apply different feeback given during the pitch of the  second week of the bootcamp

#### Description of Task to be completed?
- Correct misspelling in the API documentation
- Improve the Readme.md file

#### How should this be manually tested?
- Open the following link to see the database instruction added to readme `https://github.com/oesukam/andela-sendit`

#### What are the relevant pivotal tracker stories?
`#162459976`
